src/common/objects/CircleRigidBody.hpp

```diff
@@ -50,10 +54,20 @@ class CircleRigidBody : public GameObject {
     sf::Vector2f getTargetVelocity() const {
         return m_targetVelocity;
     }
+
     void setVelocity(sf::Vector2f velocity);
+
+    void interpolateVelocity(sf::Vector2f velocity, float a) {
+        m_position = VectorUtil::lerp(m_velocity, velocity, a);
```

Comment:
why no hermite

Reply:
hermite interpolation is only used for interpolating the position of CircleRigidBody given start pos & vel and end pos & vel